prompt-service/src/unstract/prompt_service/main.py

```diff
@@ -472,6 +481,14 @@ def prompt_processor() -> Any:
             if assertion_failed or answer.lower() == "[]" or answer.lower() == "na":
                 structured_output[output[PSKeys.NAME]] = None
             else:
+                prompt = (
+                    f"Convert the following text:\n{answer} into valid JSON format."
```

Comment:
@tahierhussain In negative and corner cases in the response where the data cannot be serialized, some info can be added to the prompt denoting the LLM to return empty/some value.